api/src/functions/getDestination.ts

```diff
@@ -0,0 +1,43 @@
+import {
+  HttpRequest,
+  InvocationContext,
+  HttpResponse,
+  HttpResponseBody,
+} from "@azure/functions";
+import axios from "axios";
+
+export async function getDestination(
+  context: InvocationContext,
+  request: HttpRequest
+): Promise<HttpResponse> {
+  const endpoint = `${process.env["SUPPLEMENTAL_API_URL"]}/destination`;
+  const authToken = request.headers.get("Authorization");
+
+  if (!authToken) {
+    context.error("Request did not contain auth token.");
+    return {
+      status: 403,
+    };
+  }
+
+  try {
+    const statusResponse = await axios({
+      url: endpoint,
+      method: "get",
+      headers: { Authorization: authToken },
+      params: request.query,
```

Comment:
Do we need to provide any query params for this request?  I don't think the destination endpoint takes any.

Reply:
You're right. This isn't needed.